astropy/utils/masked/tests/test_function_helpers.py

```diff
@@ -1462,11 +1468,7 @@ def test_nanpercentile(self):
 # Get covered functions
 tested_functions = set()
```

Comment:
So this would become
```
tested_functions = sum(get_tested_functions(cov_cls.__dict__) for cov_cls in list(...), start=set())
```
(think this works, not tested! keeping the loop and writing `tested_functions != get_tested_functions(...)` may be clearer!)